<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>780</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1196</width>
    <height>780</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>999999</width>
    <height>999999</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1200</width>
    <height>780</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QGraphicsView" name="graphicsView">
          <property name="minimumSize">
           <size>
            <width>911</width>
            <height>600</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QComboBox" name="sortBox">
            <property name="minimumSize">
             <size>
              <width>251</width>
              <height>21</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>259</width>
              <height>25</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>MergeSort</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>QuickSort</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>HeapSort</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Interpolation Sort</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sort">
            <property name="minimumSize">
             <size>
              <width>251</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>259</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>Старт</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="restart">
            <property name="minimumSize">
             <size>
              <width>251</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>259</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>Заново</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="ArraySize">
              <property name="maximumSize">
               <size>
                <width>117</width>
                <height>44</height>
               </size>
              </property>
              <property name="text">
               <string>Размер массива:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="sizeEdit">
              <property name="maximumSize">
               <size>
                <width>134</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="Time">
              <property name="minimumSize">
               <size>
                <width>71</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>126</width>
                <height>44</height>
               </size>
              </property>
              <property name="text">
               <string>Время:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>44</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="searchButton">
            <property name="minimumSize">
             <size>
              <width>251</width>
              <height>51</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>259</width>
              <height>51</height>
             </size>
            </property>
            <property name="text">
             <string>Найти</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="search">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>44</height>
               </size>
              </property>
              <property name="text">
               <string>Найти элемент:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="searchEdit">
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>26</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="index">
              <property name="minimumSize">
               <size>
                <width>71</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>126</width>
                <height>44</height>
               </size>
              </property>
              <property name="text">
               <string>Индекс:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelSearch">
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>44</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="maximumSize">
               <size>
                <width>126</width>
                <height>44</height>
               </size>
              </property>
              <property name="text">
               <string>Модуль:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="modulNumber">
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>26</height>
               </size>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>251</width>
              <height>61</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>259</width>
              <height>61</height>
             </size>
            </property>
            <property name="text">
             <string>Бинарное возведение в степень</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelSearchAfterBinPow">
            <property name="maximumSize">
             <size>
              <width>259</width>
              <height>46</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelArray">
        <property name="minimumSize">
         <size>
          <width>1181</width>
          <height>131</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
